osh.spec

```diff
@@ -232,7 +231,6 @@ fi
 %attr(640,root,root) %config(noreplace) %{_sysconfdir}/osh/worker.conf
 
 %files hub
-%defattr(-,root,apache,-)
```

Comment:
If you remove this then files will be owned by root:root.
I do NOT know what should be ownership of the files. It just seemed suspicious to me that all files in this subpackage are owned by root:apache.

Reply:
Tests in [Testing Farm](https://artifacts.dev.testing-farm.io/1b0bc17a-a514-46d3-8aab-7c9d051b6350/) passed, so I believe these changes should not break existing deployments.

Reply:
@xsuchy I was double checking this before the approval.  The files that need to be in the `apache` group use `%attr` explicitly.  Everything else is just Python scripts and configuration files, where the group can be set to `root`.